index.html

```diff
@@ -1,234 +1,216 @@
-                <h4 class="language-header"><span><img src="./images/works/Rectangle-one.svg" alt=""></span>Language</h4>
-                <ul class="skills-center">
-                    <li>Javascript</li>
-                    <li>Ruby</li>
-                    <li>Html</li>
-                    <li>Css</li>
-                </ul>
-            </div>
-            <div class="framework">
-                    <h4 class="framework-header"><span><img src="./images/works/Rectangle-two.svg" alt=""></span>Frameworks</h4>
-                    <ul class="skills-center">
-                        <li>Bootstrap</li>
-                        <li>Ruby on Rails</li>
-                        <li>RSPec</li>
-                        <li>CapyBara</li>
-                        <li>Selenium</li>
-                    </ul>
-            </div>
-            <div class="skills">
-                <h4 class="skills-header"><span><img src="./images/works/Rectangle-three.svg" alt=""></span>Skills</h4>
-                <ul class="skills-center">
-                    <li>Codekit</li>
-                    <li>Github</li>
-                    <li>Codepen</li>
-                    <li>Gitlab</li>
-                    <li>Terminal</li>
-                </ul>
-        </div>
-    </section>
-
-    <!-- Get in touch section -->
-
-<section>
-    <div class="contact-section">
-        <div class="message-box">
-            <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+        <div class="lang-skill-framework">
+          <div class="language">
+            <h1 class="language-header">
+              <span
+                ><img
+                  src="./images/Rectangle-one.svg"
+                  alt="orange diamond" /></span
+              >Language
+            </h1>
+            <ul>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </div>
+          <div class="framework">
+            <h1 class="framework-header">
+              <span
+                ><img
+                  src="./images/Rectangle-two.svg"
+                  alt="orange square" /></span
+              >Frameworks
+            </h1>
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+          <div class="skills">
+            <h1 class="skills-header">
+              <span
+                ><img
+                  src="./images/Rectangle-three.svg"
+                  alt="orange circle" /></span
+              >Skills
+            </h1>
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
```

Comment:
- [ ] It might be a good idea to fix the alignment for your technologies with the text of the about section like the Figma template. 😃 

![about section 2](https://user-images.githubusercontent.com/79876959/139561366-7c143c3f-1386-4413-ab6b-3d66be043b96.PNG)

![about section 2 Figma](https://user-images.githubusercontent.com/79876959/139561393-6ad974ec-af10-4ff6-bd08-b6a3706e9242.PNG)